sw/device/tests/closed_source/meson.build

```diff
@@ -0,0 +1,37 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# All build targets below will be build in closed source envirounment only. 
```

Comment:
nit: "will be build in closed source" --> "will be built in the closed source"
nit: "envirounment" --> "environment"